src/Utilities.jl

```diff
@@ -7,4 +7,11 @@ Utilities
 
 using Compat
 
+convert(::Type{UTF8String}, sym::Symbol)    = utf8(string(sym))
+convert(::Type{UTF8String}, s::ASCIIString) = utf8(s)
```

Comment:
Just call ``utf8`` directly. Adding methods for a function in ``Base`` to types from ``Base`` will bite us later on with hard to track bugs.